src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -330,7 +387,7 @@ Item {
     target: null
     grabPermissions: PointerHandler.CanTakeOverFromHandlersOfDifferentType | PointerHandler.ApprovesTakeOverByItems
 
-    onWheel: {
+    onWheel: function( event ) {
```

Comment:
@tomasMizera , I see you have met Qt6's wise decision to put an end to parameter injections into QML :)

Reply:
Yeah, took us a lot of time to find all the occurrences though :)